### PR TITLE
[hack] Update WMCO version to 1.0.2

### PR DESCRIPTION
**Pull request by mansikulkarni96**
This PR updates the WMCO version to 1.0.2 in hack/common.sh script.

### CODE REVIEW

**Review by aravindhp — APPROVED** [review]
/approve

**Comment by VaishnaviHire** [comment]
According to the doc `4.5` index is included as a part of this change. Just want to make sure if that is expected

**Comment by aravindhp** [comment]
/hold
Until #230 merges

File: hack/common.sh
Comment (aravindhp) [inline hack/common.sh:130]: At this point we should just update the version to 2.0.0 in master.

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**
